homeassistant/components/glances/sensor.py

```diff
@@ -223,6 +223,20 @@ class GlancesSensorEntityDescription(
         icon="mdi:docker",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ("raid", "status"): GlancesSensorEntityDescription(
+        key="status",
+        type="raid",
+        name_suffix="Raid status",
+        icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    ("raid", "type"): GlancesSensorEntityDescription(
+        key="type",
+        type="raid",
+        name_suffix="Raid type",
+        icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Similar, this returns e.g., `raid1`, right?

If so, how can this be a measurement?